elements/lisk-p2p/src/peer_directory/peer_list.ts

```diff
@@ -147,16 +147,16 @@ export class PeerList {
 
 		if (!bucket) {
 			return {
+				success: false,
 				wasPeerAdded: false,
```

Comment:
I thought `success` value already implies if a peer was added.

Reply:
I changed it back to `success` in a new commit but the definition of success is slightly different. Success doesn't mean that peer was added because if peer was added but it failed to evict while bucket was full, then that's also not success.

Reply:
I think failed to evict is an exception and should throw en error but since we are not throwing errors and handling them anywhere we can keep it as it is

Reply:
In that case should we throw an error and handle as opposed to adding another `boolean` outcome?